spec/features/invite_user_sign_up_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.